subt/ros/robot/launch/freyja.launch

```diff
@@ -5,5 +5,196 @@
     <remap from="$(arg robot_name)/image_raw/compressed" to="$(arg robot_name)/camera_front/image_raw/compressed"/>
     <remap from="$(arg robot_name)/front/depth" to="$(arg robot_name)/rgbd_front/depth"/>
     <remap from="$(arg robot_name)/points" to="$(arg robot_name)/rgbd_front/points"/>
+    <remap from="$(arg robot_name)/odom" to="$(arg robot_name)/nonexistent_input_odom"/>
```

Comment:
Can we not do this change? The odom topic is used in the old localization based on odometry and imu. I'd like to keep that so we can compare the two solutions side by side.

Reply:
We are running into troubles with the way we combine json configuration for X2 and Freyja. For Freyja, we need to propagate data from UKF, for X2, we need to keep the odoimuloc information flow path.

I.e, if we are willing to break X2 (robotika_x2 and ssci_x2) for now, we can not do this change and remove "localization.pose3d" -> "app.pose3d" from zmq-subt-x2.json.

In an ideal world, X2s would also have rtabmap, or some other improved localization, but it turns out that without the altitude sensor, the Z coordinate (and possibly other ones too) go very wrong very quickly, like z=-8 by the time X2 reaches the entrance to the cave system.

Reply:
OK, together with change removing link from OSGAR localization to app (pose3d)

Reply:
damn I missed @jisa comment :(